terraform/production/main.tf

```diff
@@ -103,57 +106,57 @@ data "aws_ssm_parameter" "mosaic_test_hostname" {
  }
 
  resource "aws_iam_role" "dms-access-for-endpoint" {
-   assume_role_policy = "${data.aws_iam_policy_document.dms_assume_role.json}"
+   assume_role_policy = data.aws_iam_policy_document.dms_assume_role.json
    name               = "dms-access-for-endpoint"
  }
 
  resource "aws_iam_role_policy_attachment" "dms-access-for-endpoint-AmazonDMSRedshiftS3Role" {
    policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonDMSRedshiftS3Role"
-   role       = "${aws_iam_role.dms-access-for-endpoint.name}"
+   role       = aws_iam_role.dms-access-for-endpoint.name
```

Comment:
Could you please remove all the IAM roles created here? They require a one-off set up and I just checked that we already have them created on prod